### ISSUE TITLE
Hard to scroll column on touch screen

### ISSUE THREAD

**Issue by PVince81** [nextcloud/deck#3799]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
Almost impossible to scroll a board column with a touch screen because:
1) either a card is under the cursor and it gets dragged at the same time as the column is scrolled
2) or the whole column is grabbed and moved
3) grabbing the very thin scrollbar doesn't work  (Firefox 99.0.1)

**To Reproduce**
1. Create a board with two columns
2. Fill the second column with enough cards to make it scroll vertically
3. Using a touch screen, try scrolling up and down

**Expected behavior**
When touching a card and scrolling up down, scroll the viewport.
Only grab the card on long press ?

@jancborchardt 

**Screenshots**

**Client details:**
 - Browser Firefox
 - Version 99.0.1
 - Device: Dell XPS 13 laptop with touchscreen, running openSUSE Tumbleweed 20220426

<details>
<summary>Server details</summary>

**Nextcloud version:** c.nc.com

**Comment by dodedodo** [comment]
These might be relevant:

#1960 is about horizontal scrolling on mobile (touchscreen).
#2480 is about horizontal scrolling on desktop. It proposes grab-n-hold scrolling that's similar to your way of scrolling.

**Comment by juliushaertl** [comment]
This s also an accessibility topic:

> Make sure the interface can be used on smaller screens without loss of content and without requiring scrolling in two directions according to the WCAG-criterion.

**Comment by chris2k20** [comment]
Any updates on that? On my oneplus 8 deck is really not usable, which is very sad because the deck's app is so a great application. Really looking forward to use that on my smartphone. ✋🏻 Thx for your time in programing that awesome software

**Comment by shoetten** [comment]
> Any updates on that? On my oneplus 8 deck is really not usable, which is very sad because the deck's app is so a great application. Really looking forward to use that on my smartphone. ✋🏻 Thx for your time in programing that awesome software

You might consider the [Android Deck App](https://github.com/stefan-niedermann/nextcloud-deck) for smartphone usage.

**Comment by libatiba** [comment]
Any updates on that? I know, it is not a payed app and users can not expect, that somebody will repair any glitches. 
Nevertheless I wonder, that an application like Nextcloud Deck is not usable on touch devices (?) (at least not iOS - I know iOS Deck App ... but it's more "alpha"...)

**Comment by hilltopmediadesign** [comment]
This should be high on the priority list. It makes deck virtually unusable on mobile.

**Comment by chris2k20** [comment]
Any updates? 
Switched to iOS and the app is not great.
